src/diff/children.js

```diff
@@ -71,7 +71,7 @@ export function diffChildren(
 				oldVNode = oldChildren[i];
 
 				if (
-					oldVNode === null ||
+					(oldVNode = oldChildren[i]) === null ||
```

Comment:
Is this `oldVNode = oldChildren[i]` necessary? There is the same declaration on line 71.

Reply:
@JoviDeCroock you are amazingly fast! 👏